From da56d8fc20c95ece57600126c8e7bc14d7a5e8c2 Mon Sep 17 00:00:00 2001
From: Rajkumar Ayyasamy <arajkuma@codeaurora.org>
Date: Thu, 28 May 2020 11:24:49 +0530
Subject: [PATCH 2/7] mtd: nand: ipq: add compatible used in linux older
 version

Added compatible string which was used in the NAND driver of linux
older version. So that u-boot can work with linux-5.4 without any changes.

Signed-off-by: Rajkumar Ayyasamy <arajkuma@codeaurora.org>
Change-Id: I2437cc37de9646fbf61864b6a328c914a6f677a7

Pick from https://git.codelinaro.org/clo/qsdk/oss/kernel/linux-ipq-5.4/-/commit/cd490bba1bbab70c69e7e8b6e8a5e026a7bd18bf

Signed-off-by: hzy <hzyitc@outlook.com>
---
 drivers/mtd/nand/raw/qcom_nandc.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mtd/nand/raw/qcom_nandc.c b/drivers/mtd/nand/raw/qcom_nandc.c
index 673468d35d28..375ae6a2d799 100644
--- a/drivers/mtd/nand/raw/qcom_nandc.c
+++ b/drivers/mtd/nand/raw/qcom_nandc.c
@@ -3042,6 +3042,10 @@ static const struct of_device_id qcom_nandc_of_match[] = {
 		.compatible = "qcom,ipq8074-nand",
 		.data = &ipq8074_nandc_props,
 	},
+	{
+		.compatible = "qcom,ebi2-nandc-bam-v1.5.0",
+		.data = &ipq8074_nandc_props,
+	},
 	{}
 };
 MODULE_DEVICE_TABLE(of, qcom_nandc_of_match);
-- 
2.25.1

